dashboard/src/components/Modal/Modal.tsx

```diff
@@ -0,0 +1,46 @@
+import { CdsButton } from "@cds/react/button";
+import { CdsModal, CdsModalActions, CdsModalContent, CdsModalHeader } from "@cds/react/modal";
+import React from "react";
+
+interface IModalProps {
+  children: JSX.Element;
+  footer?: JSX.Element;
+  hideCloseButton?: boolean;
+  modalSize?: "default" | "sm" | "lg" | "xl" | undefined;
+  showModal: boolean;
+  staticBackdrop?: boolean;
+  title?: string;
+  onModalClose: () => any;
+}
+
+function Modal({
```

Comment:
Using sometimes this modal and some other no can be confused, can we just not use the wrapper? (it's not super complex and we avoid having to write tests for this)

Reply:
Mmm, good idea, since we will only have to deal with one modal. On the other hand, this wrapper is encapsulating some cds-modal code... but, we are using other cds-modals out there... so, yes, let's get rid of the wrapper!